README.md

```diff
@@ -1,6 +1,4 @@
-## My Project
-
-TODO: Fill this README out!
+## Amazon CloudWatch Agent Test
```

Comment:
I'd prefer if this title was more descriptive. The format that I've always stuck to for README files is:
```
Title of the repo/project
------------------------
Short tag line or elevator pitch for the purpose of the repo. 

## Subheader
Stuff

...
```